cg_lims/models/sequencing_metrics.py

```diff
@@ -0,0 +1,12 @@
+from datetime import datetime
+from pydantic import BaseModel
+
+
+class SequencingMetrics(BaseModel):
```

Comment:
```suggestion
class SampleLaneSequencingMetrics(BaseModel):
```

We should keep the model name as in statusDB as to avoid confusion when working across repos in the future.